src/vitamin.cpp

```diff
@@ -69,11 +69,7 @@ void vitamin::load_vitamin( const JsonObject &jo )
         jo.throw_error( "vitamin must have a vitamin type", "vit_type" );
     }
     vit.type_ = jo.get_enum_value<vitamin_type>( "vit_type" );
-
```

Comment:
i have concerns that this will have unintended side effects.

Reply:
Looks like you put the comment on the wrong line - these are the problematic bits: <https://github.com/CleverRaven/Cataclysm-DDA/pull/41219/files#diff-73aa20e4acf5b1e1244358a4be80222dL73-L75>

Reply:
What potential sight effects come to your mind? Please elaborate.